components/shared/workCard/WorkCard.tsx

```diff
@@ -2,32 +2,55 @@ import Image from "next/image"
 import Link from "next/link"
 import React from "react"
 
+import { useGetCoverCollection } from "@/lib/cover-service-api/cover-service"
+import { getRandomContentColorClass } from "@/lib/fetchers/helper"
 import { WorkTeaserFragment } from "@/lib/graphql/generated/fbi/graphql"
+import { cn } from "@/lib/utils"
 
-import { displayCreators } from "./helper"
+import Icon from "../icon/Icon"
+import { displayCreators, getAllWorkPids, getCoverUrls } from "./helper"
 import WorkCardAvailabilityRow from "./WorkCardAvailabilityRow"
 
 type WorkCardProps = {
   work: WorkTeaserFragment
 }
 
 const WorkCard = ({ work }: WorkCardProps) => {
+  const { data } = useGetCoverCollection({
+    type: "pid",
+    identifiers: getAllWorkPids(work),
+    sizes: ["small", "large", "default", "original"],
+  })
+  const bestRepresentation = work.manifestations.bestRepresentation
+  const allPids = [bestRepresentation.pid, ...getAllWorkPids(work)]
+  const coverSrc = getCoverUrls(data, allPids || [], ["small", "large", "default", "original"])
```

Comment:
I think the cover sizes should be defined once. Either in a `const` before the `WorkCard` component but preferably in a global static variable (which there will be a system developed for in the near future).
I suggest a TODO message saying that cover sizes will be handled in a global configuration in the future